Library/Homebrew/utils.rb

```diff
@@ -187,7 +187,23 @@ def system(cmd, *args)
     _system(cmd, *args)
   end
 
+  def install_and_require_rubocop!
+    install_gem_setup_path! "bundler"
+    require "bundler"
+
+    quiet_system "bundle", "config", "--delete", "without"
+    quiet_system "bundle", "config", "--local", "without", "test"
```

Comment:
Will this not uninstall the test gems meaning `brew tests && brew style` always cause a `bundle install` to be needed? I could be totally wrong here.

Reply:
The `test` ones will still be installed.

Reply:
Ok.